src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceCloseToBlock.java

```diff
@@ -0,0 +1,37 @@
+/*
+VariableDeclarationUsageDistance
+allowedDistance = 1
+ignoreVariablePattern = (default)
+validateBetweenScopes = false
+ignoreFinal = false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.variabledeclarationusagedistance;
+
+public class InputVariableDeclarationUsageDistanceCloseToBlock {
+    void method() {
+        int a = 3; // violation 'Distance between .* declaration and its first usage is 13.'
+        int b = 2; // violation 'Distance between .* declaration and its first usage is 13.'
```

Comment:
Add example in this same test showing what the code looks like with no violations.

Reply:
@rnveach done

Reply:
Not done. The code before and after looks nothing a like. I am expecting a fix of the original code in the issue and what it will look like in the future with no violations as if the developer fixed it himself.

Reply:
@rnveach please check the updated example